api/models/call.go

```diff
@@ -141,6 +141,12 @@ type Call struct {
 
 	// Fn this call belongs to.
 	FnID string `json:"fn_id" db:"fn_id"`
+
+	// Image Pull Duration
+	PullStartedAt common.DateTime `json:"pull_started_at" db:"pull_started_at"`
+
+	// Image Pull Duration
+	PullEndedAt common.DateTime `json:"pull_ended_at" db:"pull_ended_at"`
```

Comment:
we don't need to add these to model/call. We can use the same hackery we have used for userExecTime in agent/call.go (https://github.com/fnproject/fn/blob/master/api/agent/call.go#L295)

Reply:
Agreed, I'll change it up

Reply:
https://github.com/fnproject/fn/blob/8cef854cccf96da4d78cf1aa6d0dd8b478a8fe8b/api/models/call.go#L127